apps/admin_app/lib/admin_app_web/helpers.ex

```diff
@@ -1,12 +1,8 @@
 defmodule AdminAppWeb.Helpers do
   import Ecto.Changeset
-  import Ecto.Query
-  alias Ecto.Adapters.SQL
+
   alias Snitch.Core.Tools.MultiTenancy.Repo
-  alias Snitch.Data.Schema.Order
-  alias Snitch.Domain.Order, as: Domain
-  alias Elixlsx.{Workbook, Sheet}
-  alias AdminAppWeb.DataExportMail
+  alias Snitch.Data.Schema.{Order, Product}
 
   @months ["Jan", "Feb", "Mar", "Apr", "May", "June", "July", "Aug", "Sept", "Oct", "Nov", "Dec"]
```

Comment:
Do we really need months like these? @jyotigautam can't it be somehow be fetched from `Calendar` module or something. Names of months is a pretty generic thing to define as a module attribute in my opinion.

Reply:
No, we need to  have a list of month names since the Calendar module or any other module dealing with date time returns month as a numerical value(1-12)

Reply:
We cab use strftime: https://hexdocs.pm/timex/Timex.Format.DateTime.Formatters.Strftime.html
What is the usecase? Do we need to format full date and time? Or find name of month for a specific month number ?

Reply:
We can format whole date using strftime. Need to check how to get month name using this: https://hexdocs.pm/timex/Timex.Format.DateTime.Formatters.Strftime.html#module-months

Reply:
sir i found out this ...Elixir.Timex.month_shortname(1)

Reply:
@suhasdombale123 I appreciate your proactive response! Keep it up.